### PR TITLE
docs/build: document setup with the BeagleBone Black

### PR DESCRIPTION
**Pull request by chombourger**
Signed-off-by: Cedric Hombourger <cedric.hombourger@siemens.com>

### CODE REVIEW

File: docs/build.rst [hunk 1]
```diff
@@ -199,6 +199,135 @@ and install it::
...
 enables QEMU's support for TPM devices.
 
+BeagleBone Black
+----------------
+
+The BeagleBone Black (abbreviated as BBB) is a low-cost, community-supported
+development platform for developers and hobbyists.
+
+It is powered by an AM335x Cortex-A8 processor, has a 4GiB eMMC, a microSD slot,
+a microUSB OTG port, a USB-Host Type A port, an Ethernet port and GPIO pins.
```
Comment (bhargavdas) [inline docs/build.rst:209]: miniUSB OTG port

File: docs/build.rst [hunk 2]
```diff
@@ -199,6 +199,135 @@ and install it::
...
+      of=/dev/mmcblk0 bs=8M
+
+(replace ``/dev/mmcblk0`` with the actual SD card device on your system).
+
+Booting the BeagleBone Black
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Insert the microSD card created above into the microSD card slot of your
+BeagleBone Black and connect the board to your network. Attach a formatted
+USB stick to the USB-Host port. Lastly, get a microUSB cable, connect your
```
Comment (bhargavdas) [inline docs/build.rst:251]: Change
"Lastly, get a microUSB cable, connect your"
to 
"Lastly, get a miniUSB cable, connect your"

**Comment by chombourger** [comment]
We still need to add BBB diagrams to complete this PR

**Comment by chombourger** [comment]
Done with #327

**Comment by bhargavdas** [comment]
@chombourger we missed this commit 
https://github.com/chombourger/mtda/pull/1/commits/043e329e1c10401a6ff705200c88c569da3057f8

**Pull request closed without merging**